scripts/operation/start-trubudget.sh

```diff
@@ -53,7 +53,6 @@ ENABLED_SERVICES=""
 SLAVE_SERVICES=""
 IS_FULL=false
 HAS_SLAVE=false
-HAS_FRONTEND_LOGGING=false
```

Comment:
bad smell. refactored so the logging-service is treated like every other service in this bash script